server/resolvers/AtVersion/index.js

```diff
@@ -0,0 +1,7 @@
+const supportedByAutomationResolver = require('./supportedByAutomationResolver');
+
+const AtVersion = {
+  supportedByAutomation: supportedByAutomationResolver
```

Comment:
nit: just for consistency, the other files in `/resolver` import the resolver as the expect attribute name, so `const supportedByAutomation = require('./supportedByAutomationResolver');`

Reply:
Good call!